bottom-nav.

MossbruggerLu opened the pull request:
added bottom-nav.

stritti committed:
add package-lock for required 'npm ci'

chrisWolter commented:
Könnt Ihr dem Content-Slot (<router-view/>) oder dessem Inhalt noch einen Margin geben, damit der Inhalt nicht hinter dem Fixed Footer verschwindet?

Commit:
bottom-nav.

added bottom-nav.

Commit:
updated router-view

addded margin bottom of 80px to ensure the bottom navgation does not overlappe the router view.
